packages/cdk/scroll/src/virtual/composables/useScrollState.ts

```diff
@@ -73,7 +66,7 @@ function calcState(
 ) {
   const dataLength = dataSource.length
   // Always use virtual scroll bar in avoid shaking
-  if (!virtual || dataLength === 0 || itemHeight * dataLength <= height) {
+  if (!virtual || dataLength === 0 || (!virtual && itemHeight * dataLength <= height)) {
```

Comment:
```suggestion
  if (!virtual || dataLength === 0 || (itemHeight * dataLength <= height)) {
```
这个条件没任何意义啊，!virtual 在前面就判断了。